pontoon/allauth_urls.py

```diff
@@ -6,12 +6,29 @@
 
 import importlib
 
+from typing import List, Union
+
 from allauth.account import views as account_views
 from allauth.socialaccount import providers, views as socialaccount_views
 
 from django.conf import settings
 from django.contrib.auth import views
-from django.urls import path
+from django.urls import URLPattern, URLResolver, path
+
+
+def build_provider_urlpatterns() -> List[Union[URLPattern, URLResolver]]:
+    provider_urlpatterns: List[Union[URLPattern, URLResolver]] = []
+    provider_classes = providers.registry.get_class_list()
+
+    provider_classes = [
+        cls for cls in provider_classes if cls.id != "openid_connect"
+    ] + [cls for cls in provider_classes if cls.id == "openid_connect"]
+    for provider_class in provider_classes:
+        prov_mod = importlib.import_module(provider_class.get_package() + ".urls")
+        prov_urlpatterns = getattr(prov_mod, "urlpatterns", None)
+        if prov_urlpatterns:
+            provider_urlpatterns += prov_urlpatterns
+    return provider_urlpatterns
```

Comment:
Can we import `build_provider_urlpatterns()` from the library rather than copying it?